### ISSUE TITLE
NTR: Parainfluenza virus RT-PCR in blood or sputum positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8090]
New term request Parainfluenza virus RT-PCR in blood or sputum positive (nadea3_190227153855): Paraflu positive; Reverse transcriptase polymerase chain reaction; RNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000738
- label: Positive bloodstream parainfluenza virus nucleic acid test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of parainfluenza virus in the blood circulation by a methiod such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8090
- synonyms:
